BUILDING.md

```diff
@@ -35,6 +35,8 @@ Key arguments to build.cmd / build:
  * -ShowDescription                   Shows all of the build tasks and their descriptions
  * -Experimental, -e                  Use the experimental build of Roslyn
 
+Note that the above format for supplying arguments only work on Windows. On Linux one has to use double dashes to specify the argument, like `build.sh --target=Test` or `build.sh --target Test` to run the `Test` task.
+
```

Comment:
Unless, the current directory is on the path, you need './build.sh' in the distro I have used.

Reply:
Corrected